### ISSUE TITLE
The new approach (in 3.0.0-beta1) broke it for PoweShell 5?

### ISSUE THREAD

**Issue by nohwnd** [nohwnd/Profiler#14]

**Comment by fflaten** [comment]
At least something broke, badly.

```powershell
[23:15:50] C:\git\Profiler> $t = Trace-Script { & .\demo-scripts\t.ps1 }
Running in PowerShell 5.1.19041.906.
Tracing...
Run  failed after 00:00:00.0118656 with Exception calling "Unpatch" with "0" argument(s): "Object reference not set to an instance of an object.".
Tracing done. Got 0 trace events.
Processing 0 trace events.
Cannot index into a null array.
At C:\git\Profiler\Profiler\Trace-Script.ps1:156 char:24
+         foreach ($f in $Trace[1..($Trace.Length - 2)]) {
+                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : NullArray

Loading sources for 0 files.
Figuring out flow.
....
```

**Comment by nohwnd** [comment]
The field was renamed to have _ in v7. Simple fix.

**Comment by nohwnd** [comment]
Fixed in #21
